scripts/download-readmes.js

```diff
@@ -0,0 +1,167 @@
+      'babel-plugin-transform-member-expression-literals',
+      'babel-plugin-transform-object-assign',
+      'babel-plugin-transform-object-rest-spread',
+      'babel-plugin-transform-object-set-prototype-of-to-assign',
+      'babel-plugin-transform-proto-to-assign',
+      'babel-plugin-transform-react-constant-elements',
+      'babel-plugin-transform-react-display-name',
+      'babel-plugin-transform-react-inline-elements',
+      'babel-plugin-transform-react-jsx-compat',
+      'babel-plugin-transform-react-jsx-self',
+      'babel-plugin-transform-react-jsx-source',
+      'babel-plugin-transform-react-jsx',
+      'babel-plugin-transform-regenerator',
+      'babel-plugin-transform-runtime',
+      'babel-plugin-transform-strict-mode',
+      'babel-plugin-undeclared-variables-check',
+      'babel-core',
+      'babel-register',
+      'babel-cli',
+      'babel-polyfill',
+      'babel-register',
+    ]
+  }, {
+    repo: 'babili',
+    packages: [
+      'babel-plugin-minify-constant-folding',
+      'babel-plugin-minify-dead-code-elimination',
+      'babel-plugin-minify-flip-comparisons',
+      'babel-plugin-minify-guarded-expressions',
+      'babel-plugin-minify-infinity',
+      'babel-plugin-minify-mangle-names',
+      'babel-plugin-minify-numeric-literals',
+      'babel-plugin-minify-replace',
+      'babel-plugin-minify-simplify',
+      'babel-plugin-minify-type-constructors',
+      'babel-plugin-transform-inline-consecutive-adds',
+      'babel-plugin-transform-inline-environment-variables',
+      'babel-plugin-transform-merge-sibling-variables',
+      'babel-plugin-transform-minify-booleans',
+      'babel-plugin-transform-node-env-inline',
+      'babel-plugin-transform-property-literals',
+      'babel-plugin-transform-regexp-constructors',
+      'babel-plugin-transform-remove-console',
+      'babel-plugin-transform-remove-debugger',
+      'babel-plugin-transform-simplify-comparison-operators',
+      'babel-plugin-transform-undefined-to-void',
+    ],
+  }, {
+    repo: 'babel-preset-env',
+    packages: [
+      'babel-preset-env',
+    ],
+    customURIs: {
+      'babel-preset-env': '/README.md',
+    },
+  }, {
+    repo: 'babylon',
+    packages: [
+      'babylon',
+    ],
+    customURIs: {
+      'babylon': '/README.md',
+    },
+  },
+];
+const numConcurrentRequests = 20;
+
+console.log('Downloading READMEs');
+packageReadmes.forEach(repo => {
+  async.mapLimit(repo.packages, numConcurrentRequests, (packageName, cb) => {
+    const uri = repo.customURIs && repo.customURIs[packageName]
+      ? repo.customURIs[packageName]
+      : `/packages/${packageName}/README.md`;
+    const readmeURL = `https://raw.githubusercontent.com/babel/${repo.repo}/master${uri}`;
+
+    fetch(readmeURL)
+      .then(res => res.text())
+      .then(text => {
+        fs.writeFile(`${__dirname}/../_includes/readmes/${packageName}.md`, text, cb);
+        console.log(`- ${packageName}`);
+      });
+  });
+});
```

Comment:
Just an idea but you can actually list the directories through the GitHub API:

```
curl https://api.github.com/repos/babel/babel/contents/packages
curl https://api.github.com/repos/babel/babili/contents/packages
``` 

It returns all the directories and a few files in the `packages` folder. GitHub API is rate limited but I think it will be fine.